lib/components/narrative/default/default-itinerary.js

```diff
@@ -148,12 +143,36 @@ const ITINERARY_ATTRIBUTES = [
         // FIXME: For CAR mode, walk time considers driving time.
         <>
           <FormattedDuration duration={itinerary.walkTime} />
-          <LegIconWrapper>
+          <LegIconWrapper noSpace>
             <LegIcon leg={leg} size={5} />
           </LegIconWrapper>
         </>
       )
     }
+  },
+  {
+    id: 'carbonEmission',
+    order: 4,
+    render: (itinerary, { co2Config }) => {
+      if(!co2Config) return // If there is no CO2 Config, don't display this element
```

Comment:
Would it be possible to add the feature we were talking about where the value is only shown if there is a significant difference (and then making the results really green and really red perhaps based on how big of a difference there is)? These rows are getting really tall... and 15oz vs 14oz is not always the most helpful info...

Reply:
We have the percentages/relative values, as well as some rounding and hiding values below 10%. Does this feel like enough? The design is not great, but it really just needs a complete redesign. I think adding bright colors would be a bit too much.